app/screens/Login/Login.js

```diff
@@ -0,0 +1,42 @@
+import React, { Component } from 'react';
+import { Text, View, Image, TextInput, TouchableHighlight } from 'react-native';
+
+import { iconImages } from './../../images';
+
+import { styles } from './styles';
+
+class Login extends Component {
+    state = {
+        inputTextEmail: 'email',
+        inputTextBox: 'Text box'
+    };
+
+    render() {
+        return (
+            <View style={styles.container}>
+                <Image
+                    source={iconImages.cookieMonster}
+                    style={styles.image}/>
+                <Text style={styles.title}>Friday's shop</Text>
+                <TextInput
+                    style={styles.input}
+                    onChangeText={(inputTextEmail) => this.setState({inputTextEmail})}
```

Comment:
It's not good practice to create function in render. But I'm sure that you have already knew it ;)